spec/features/admin/gdpr/requests_spec.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe 'GDPR requests', type: :feature, js: true do
+  include Warden::Test::Helpers
+
+  let(:admin_user) { create(:admin_user) }
+
+  before { login_as admin_user }
+
+  context 'when there are not requests' do
```

Comment:
You should either write _when there aren't any requests_ or _when there are no requests_, which is a bit more colloquial but still widely accepted (it's what I would write).